config/vulcan.default.yml

```diff
@@ -49,3 +49,4 @@ test:
   <<: *defaults
 production:
   <<: *defaults
+ 
```

Comment:
Can you make sure that you have "Insert Final Newline" enabled and you also have "Trim trailing whitespace" enabled in your editor? It looks like this file and `docs/_config.yml` are showing no newline at end of file.